extensions/value_generators/models/generators.py

```diff
@@ -31,7 +30,7 @@ class Copier(value_generators_domain.BaseValueGenerator):
 
     default_value = ''
 
-    def generate_value(self, context_params, value, parse_with_jinja=False):
+    def generate_value(self, context_params, value, parse_with_jinja=False):  # pylint: disable=unused-argument
```

Comment:
can we use unused_ prefix and remove the lint bypass flag?

Reply:
Done